ui/app/templates/vault/cluster.hbs

```diff
@@ -9,16 +10,13 @@
       <ul class="navbar-sections {{if (has-feature "Namespaces") 'with-ns-picker'}}">
         {{#if (has-feature "Namespaces")}}
           <li>
-            <NamespacePicker
-              @class="navbar-item"
-              @namespace={{namespaceQueryParam}}
-            >
+            <NamespacePicker @class="navbar-item" @namespace={{namespaceQueryParam}}>
```

Comment:
I'm guessing this is a VSCode auto-formatting thing, but I think the previous formatting is easier to read and edit. I remember having to turn off VSCode formatting because it was trying to do too much (but I also didn't try too hard to make it work).